### ISSUE TITLE
Modal actions aren't intuitive

### ISSUE THREAD

**Issue by FichteFoll** [FluffyStuff/OpenRiichi#31]
I've just played a round with 2 others and both of them had issues with the riichi and chi buttons seeming unresponsive when most likely they had to make a choice for building a meld or discarding a tile. However, the UI didn't do a good job in reflecting that.

I know from other implementations (mobile, web) that they highlight the tiles that can be thrown out and for chi they provide the exact options to build a meld that to choose from. I believe this to be much better UX.

**Comment by band-a-prend** [comment]
I agree. When I see "Riichi" I can't understand how to choose tile to discard and to announce Riichi.
Especially if I already was confused and drop "red five" I don't understand how to announce Riichi again with drop to discard another common "five" tile.
